### PR TITLE
MAID-1262.

### PR DESCRIPTION
**Pull request by s-brian**
[Review on Reviewable](https://reviewable.io/reviews/maidsafe/routing/485)

### CODE REVIEW

**Comment by maidsafe-highfive** [comment]
r? @benjaminbollen

(maidsafe_highfive has picked a reviewer for you, use r? to override)

File: src/routing_membrane.rs [hunk 1]
```diff
@@ -361,6 +361,7 @@ impl<F> RoutingMembrane<F> where F: Interface {
 
     /// When CRUST reports a lost connection, ensure we remove the endpoint anywhere
     /// TODO: A churn event might be triggered
+    #[allow(unused_variables)]
```
Comment (benjaminbollen) [inline src/routing_membrane.rs]: starting a variable with an _ allows for that variable to be unused

Comment (s-brian) [inline src/routing_membrane.rs]: Just added that to the PR.

File: src/routing_membrane.rs [hunk 2]
```diff
@@ -720,6 +715,7 @@ impl<F> RoutingMembrane<F> where F: Interface {
         }
     }
 
+    #[allow(unused_variables)]
```
Comment (benjaminbollen) [inline src/routing_membrane.rs]: again, better to not name or _name unused variables; thanks

File: src/routing_membrane.rs [hunk 3]
```diff
@@ -950,25 +952,29 @@ impl<F> RoutingMembrane<F> where F: Interface {
...
+            match method_call {
+                MethodCall::Put { destination: x, content: y, } => self.put(x, y),
+                MethodCall::Get { name: x, data_request: y, } => self.get(x, y),
+                MethodCall::Refresh { type_tag, from_group, payload } => self.refresh(type_tag, from_group, payload),
+                MethodCall::Post { destination: x, content: y, } => self.post(x, y),
+                MethodCall::Delete { name: x, data : y } => self.delete(x, y),
+                MethodCall::None => (),
+                MethodCall::Forward { destination } =>
+                    ignore(self.forward(&signed_message, &message, destination)),
+                MethodCall::Reply { data } =>
```
Comment (benjaminbollen) [inline src/routing_membrane.rs]: this should probably not be supported, "put_response reply" as data is discarded

File: src/routing_membrane.rs [hunk 4]
```diff
@@ -1228,23 +1235,29 @@ impl<F> RoutingMembrane<F> where F: Interface {
...
+            match method_call {
+                MethodCall::Put { destination: x, content: y, } => self.put(x, y),
+                MethodCall::Get { name: x, data_request: y, } => self.get(x, y),
+                MethodCall::Refresh { type_tag, from_group, payload } => self.refresh(type_tag, from_group, payload),
+                MethodCall::Post { destination: x, content: y, } => self.post(x, y),
+                MethodCall::Delete { name: x, data : y } => self.delete(x, y),
+                MethodCall::None => (),
+                MethodCall::Forward { destination } =>
+                    ignore(self.forward(&orig_message, &message, destination)),
+                MethodCall::Reply { data } =>
```
Comment (benjaminbollen) [inline src/routing_membrane.rs]: again, on get_data_response, reply should not be supported

**Comment by benjaminbollen** [comment]
please look at the other comments that are now on "an outdated diff"; they are still relevant

**Pull request merged**